Разметка всех страниц

Yarilkins opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2167205/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/2167205-sedona-36/2/

Yarilkins committed:
Добавляет header body footer

baronin commented on index.html:
```diff
@@ -5,8 +5,140 @@
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
-
-  </body>
+    <header class="page_header">
```
Почему все классы через нижнее подчеркивание?

baronin commented on index.html:
```diff
@@ -5,8 +5,140 @@
+    </header>
+    <main class="main_index">
+        <h1 class="visually_hidden">Cedona</h1>
+        <img src="#" alt="Welcome">
```
Это не просто картинка,a полноценная секция, обычно названия им дают hero или intro
![image](https://user-images.githubusercontent.com/20131942/213934033-a9cfb020-a1f1-42b4-9dc5-12b4448a99a6.png)

baronin reviewed (changes requested):
- html код не отформатирован
- Нет разметки страницы catalog
- Нет сеции "Приезжайте..."
![image](https://user-images.githubusercontent.com/20131942/213934178-2fe69b8e-fa57-4814-b462-76b9a85da7a8.png)

Yarilkins committed:
Исправляет замечания

Форматирует текст, меняет "_" на "-" в названиях классов.
добавляет section с классом intro
добавляет секцию с facilities

Yarilkins commented:
Исправляет замечания, добавляет секции и меняет нижние подчеркивания на тире

Yarilkins commented on index.html:
```diff
@@ -5,8 +5,149 @@
+        </section>
+        <section class="newslatter">
+            <h2 class="newslatter-title">Подпишитесь на рассылку</h2>
+            <p class="newslatter-notation">Только полезная информация и никакого спама, честное бойскаутское!</p>
```
я так понимаю, что в данной ситуации, в этой секции задний фон будет добавляться не через <img>, а через css-свойство background?
